src/main/resources/db/migration/all/20241128163133__application_controller_and_linked_services_in_cas2bail.sql

```diff
@@ -0,0 +1,157 @@
+
+CREATE TABLE cas_2_bail_application_json_schemas
+(
+    json_schema_id UUID NOT NULL,
+    CONSTRAINT pk_cas_2_bail_application_json_schemas PRIMARY KEY (json_schema_id)
+);
+
+CREATE TABLE cas_2_bail_application_notes
+(
+    id                          UUID NOT NULL,
+    application_id              UUID,
+    created_at                  TIMESTAMP WITHOUT TIME ZONE,
```

Comment:
we use timestamptz everywhere so should do the same for all entries here

It's also not recommended by postgres - https://wiki.postgresql.org/wiki/Don't_Do_This#Don.27t_use_timestamp_.28without_time_zone.29

Reply:
Yes done